app/config/CTCServicesConfig.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package config
+
+import com.google.inject.Inject
+import play.api.Configuration
+import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
+
+/*
+ * This class exists simply to expose the config function, which allows us to
+ * avoid a magic string already present in the ServicesConfig object.
+ */
+class CTCServicesConfig @Inject() (config: Configuration) extends ServicesConfig(config) {
+
+  override def config(serviceName: String): Configuration = super.config(serviceName)
+
+}
```

Comment:
[I added a comment to the class to explain it](https://github.com/hmrc/common-transit-convention-traders/pull/314/files/47fc965442421acf5ecde83b660bc0bd4adec5af#diff-3ebf2055d88ed788e8aa3a1b7f1f8cea47035585bba0fd3e75a0c82de3ff5a22R23-R26) -- it's to get access to the protected method `config` so that we don't have to replicate getting the config for a specific key.